### ISSUE TITLE
"cpan -i Sys::CPU" failed on Cygwin

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#13094]
Migrated from [rt.perl.org#118819](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=118819) (status was 'rejected')

Searchable as RT118819$

**Comment by p5pRT** [comment]
### From tuyenqh@yahoo.com

Subject&#8203;: "cpan \-i Sys&#8203;::CPU" failed on Cygwin
Message\-Id&#8203;: \<5\.14\.2\_19888\_1373366884@&#8203;AZHA1>
Reply\-To&#8203;: tuyenqh@&#8203;yahoo\.com
To&#8203;: perlbug@&#8203;perl\.org
From&#8203;: tuyenqh@&#8203;yahoo\.com


This is a bug report for perl from tuyenqh@&#8203;yahoo\.com\,
generated with the help of perlbug 1\.39 running under perl 5\.14\.2\.

On my Cyswin

cpan \-i Sys&#8203;::CPU failed with following output
The gcc version is 4\.7\.3

Regards\,
Tuyen
=======================================================================


perl&#8203;: warning&#8203;: Setting locale failed\.
perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
LC\_ALL = \(unset\)\,
LANG = "DA"
    are supported and installed on your system\.
perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
CPAN&#8203;: Storable loaded ok \(v2\.27\)
Going to read '/home/tqhoang/\.cpan/Metadata'
  Database was generated on Mon\, 08 Jul 2013 06&#8203;:07&#8203;:29 GMT
Running install for module 'Sys&#8203;::CPU'
Running make for M/MZ/MZSANFORD/Sys\-CPU\-0\.54\.tar\.gz
CPAN&#8203;: Digest&#8203;::SHA loaded ok \(v5\.71\)
CPAN&#8203;: Compress&#8203;::Zlib loaded ok \(v2\.033\)
Checksum for /home/tqhoang/\.cpan/sources/authors/id/M/MZ/MZSANFORD/Sys\-CPU\-0\.54\.tar\.gz ok
CPAN&#8203;: File&#8203;::Temp loaded ok \(v0\.22\)
CPAN&#8203;: Parse&#8203;::CPAN&#8203;::Meta loaded ok \(v1\.4401\)
CPAN&#8203;: CPAN&#8203;::Meta loaded ok \(v2\.110440\)
CPAN&#8203;: Module&#8203;::CoreList loaded ok \(v2\.49\_02\)

  CPAN\.pm&#8203;: Going to build M/MZ/MZSANFORD/Sys\-CPU\-0\.54\.tar\.gz

perl&#8203;: warning&#8203;: Setting locale failed\.
perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
LC\_ALL = \(unset\)\,
LANG = "DA"
    are supported and installed on your system\.
perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
Checking if your kit is complete\.\.\.
Looks good
Writing Makefile for Sys&#8203;::CPU
Writing MYMETA\.yml
perl&#8203;: warning&#8203;: Setting locale failed\.
perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
LC\_ALL = \(unset\)\,
LANG = "DA"
    are supported and installed on your system\.
perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
perl&#8203;: warning&#8203;: Setting locale failed\.
perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
LC\_ALL = \(unset\)\,
LANG = "DA"
    are supported and installed on your system\.
perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
perl&#8203;: warning&#8203;: Setting locale failed\.
perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
LC\_ALL = \(unset\)\,
LANG = "DA"
    are supported and installed on your system\.
perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
perl&#8203;: warning&#8203;: Setting locale failed\.
perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
LC\_ALL = \(unset\)\,
LANG = "DA"
    are supported and installed on your system\.
perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
perl&#8203;: warning&#8203;: Setting locale failed\.
perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
LC\_ALL = \(unset\)\,
LANG = "DA"
    are supported and installed on your system\.
perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
perl&#8203;: warning&#8203;: Setting locale failed\.
perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
LC\_ALL = \(unset\)\,
LANG = "DA"
    are supported and installed on your system\.
perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
perl&#8203;: warning&#8203;: Setting locale failed\.
perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
LC\_ALL = \(unset\)\,
LANG = "DA"
    are supported and installed on your system\.
perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
perl&#8203;: warning&#8203;: Setting locale failed\.
perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
LC\_ALL = \(unset\)\,
LANG = "DA"
    are supported and installed on your system\.
perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
perl&#8203;: warning&#8203;: Setting locale failed\.
perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
LC\_ALL = \(unset\)\,
LANG = "DA"
    are supported and installed on your system\.
perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
cp CPU\.pm blib/lib/Sys/CPU\.pm
/usr/bin/perl\.exe /usr/lib/perl5/5\.14/ExtUtils/xsubpp  \-typemap /usr/lib/perl5/5\.14/ExtUtils/typemap  CPU\.xs > CPU\.xsc && mv CPU\.xsc CPU\.c
perl&#8203;: warning&#8203;: Setting locale failed\.
perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
LC\_ALL = \(unset\)\,
LANG = "DA"
    are supported and installed on your system\.
perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
Please specify prototyping behavior for CPU\.xs \(see perlxs manual\)
gcc\-4 \-c    \-DPERL\_USE\_SAFE\_PUTENV \-U\_\_STRICT\_ANSI\_\_ \-g \-fno\-strict\-aliasing \-pipe \-fstack\-protector \-DUSEIMPORTLIB \-O3    \-DVERSION=\\"0\.54\\"  \-DXS\_VERSION=\\"0\.54\\"  "\-I/usr/lib/perl5/5\.14/i686\-cygwin\-threads\-64int/CORE"   CPU\.c
CPU\.xs&#8203;: In function 'XS\_Sys\_\_CPU\_cpu\_clock'&#8203;:
CPU\.xs&#8203;:349&#8203;:5&#8203;: error&#8203;: unknown type name 'processor\_info\_t'
CPU\.xs&#8203;:350&#8203;:48&#8203;: error&#8203;: request for member 'pi\_state' in something not a structure or union
CPU\.xs&#8203;:350&#8203;:62&#8203;: error&#8203;: 'P\_ONLINE' undeclared \(first use in this function\)
CPU\.xs&#8203;:350&#8203;:62&#8203;: note&#8203;: each undeclared identifier is reported only once for each function it appears in
CPU\.xs&#8203;:351&#8203;:26&#8203;: error&#8203;: request for member 'pi\_clock' in something not a structure or union
CPU\.xs&#8203;:352&#8203;:26&#8203;: error&#8203;: request for member 'pi\_clock' in something not a structure or union
CPU\.xs&#8203;: In function 'XS\_Sys\_\_CPU\_cpu\_type'&#8203;:
CPU\.xs&#8203;:389&#8203;:5&#8203;: error&#8203;: unknown type name 'processor\_info\_t'
CPU\.xs&#8203;:391&#8203;:16&#8203;: error&#8203;: request for member 'pi\_processor\_type' in something not a structure or union
Makefile&#8203;:329&#8203;: recipe for target \`CPU\.o' failed
make&#8203;: \*\*\* \[CPU\.o\] Error 1
  MZSANFORD/Sys\-CPU\-0\.54\.tar\.gz
  /usr/bin/make \-\- NOT OK
'YAML' not installed\, will not store persistent state
Running make test
  Can't test without successful make
Running make install
  Make had returned bad status\, install seems impossible


---
Flags&#8203;:
    category=library
    severity=low


---
Site configuration information for perl 5\.14\.2&#8203;:

Configured by rurban at Thu Jul 12 13&#8203;:58&#8203;:56 CDT 2012\.

Summary of my perl5 \(revision 5 version 14 subversion 2\) configuration&#8203;:
   
  Platform&#8203;:
    osname=cygwin\, osvers=1\.7\.15\(0\.26053\)\, archname=cygwin\-thread\-multi\-64int
    uname='cygwin\_nt\-5\.1 winxp 1\.7\.15\(0\.26053\) 2012\-05\-09 10&#8203;:25 i686 cygwin '
    config\_args='\-de \-Dlibperl=cygperl5\_14\.dll \-Dcc=gcc\-4 \-Dld=g\+\+\-4 \-Darchname=i686\-cygwin\-threads\-64int \-Dmksymlinks \-Dusethreads \-Accflags=\-g'
    hint=recommended\, useposix=true\, d\_sigaction=define
    useithreads=define\, usemultiplicity=define
    useperlio=define\, d\_sfio=undef\, uselargefiles=define\, usesocks=undef
    use64bitint=define\, use64bitall=undef\, uselongdouble=undef
    usemymalloc=n\, bincompat5005=undef
  Compiler&#8203;:
    cc='gcc\-4'\, ccflags ='\-DPERL\_USE\_SAFE\_PUTENV \-U\_\_STRICT\_ANSI\_\_ \-g \-fno\-strict\-aliasing \-pipe \-fstack\-protector'\,
    optimize='\-O3'\,
    cppflags='\-DPERL\_USE\_SAFE\_PUTENV \-U\_\_STRICT\_ANSI\_\_ \-g \-fno\-strict\-aliasing \-pipe \-fstack\-protector'
    ccversion=''\, gccversion='4\.5\.3'\, gccosandvers=''
    intsize=4\, longsize=4\, ptrsize=4\, doublesize=8\, byteorder=12345678
    d\_longlong=define\, longlongsize=8\, d\_longdbl=define\, longdblsize=12
    ivtype='long long'\, ivsize=8\, nvtype='double'\, nvsize=8\, Off\_t='off\_t'\, lseeksize=8
    alignbytes=8\, prototype=define
  Linker and Libraries&#8203;:
    ld='g\+\+\-4'\, ldflags =' \-Wl\,\-\-enable\-auto\-import \-Wl\,\-\-export\-all\-symbols \-Wl\,\-\-enable\-auto\-image\-base \-fstack\-protector \-L/usr/local/lib'
    libpth=/usr/local/lib /usr/lib /lib
    libs=\-lgdbm \-ldb \-ldl \-lcrypt \-lgdbm\_compat
    perllibs=\-ldl \-lcrypt
    libc=/usr/lib/libc\.a\, so=dll\, useshrplib=true\, libperl=cygperl5\_14\.dll
    gnulibc\_version=''
  Dynamic Linking&#8203;:
    dlsrc=dl\_dlopen\.xs\, dlext=dll\, d\_dlsymun=undef\, ccdlflags=' '
    cccdlflags=' '\, lddlflags=' \-\-shared  \-Wl\,\-\-enable\-auto\-import \-Wl\,\-\-export\-all\-symbols \-Wl\,\-\-enable\-auto\-image\-base \-L/usr/local/lib \-fstack\-protector'

Locally applied patches&#8203;:
    Bug\#55162 File&#8203;::Spec&#8203;::case\_tolerant performance
    CYG07 $vendorarch/auto/\.rebase
    CYG15 static Win32CORE
    CYG17 cyg\-1\.7 paths\-utf8
    0c612ce82 Fix building static extensions on cygwin\, \-UUSEIMPORTLIB
    1bac5ecc1 Fix 64\-bit threading sv\.c&#8203;: S\_anonymise\_cv\_maybe
    Cygwin&#8203;::sync\_winenv added



---
@&#8203;INC for perl 5\.14\.2&#8203;:
    /usr/lib/perl5/site\_perl/5\.14/i686\-cygwin\-threads\-64int
    /usr/lib/perl5/site\_perl/5\.14
    /usr/lib/perl5/vendor\_perl/5\.14/i686\-cygwin\-threads\-64int
    /usr/lib/perl5/vendor\_perl/5\.14
    /usr/lib/perl5/5\.14/i686\-cygwin\-threads\-64int
    /usr/lib/perl5/5\.14
    /usr/lib/perl5/site\_perl/5\.10
    /usr/lib/perl5/vendor\_perl/5\.10
    /usr/lib/perl5/site\_perl/5\.8
    \.



---
Environment for perl 5\.14\.2&#8203;:
    HOME=/home/tqhoang
    LANG=en\_US\.UTF\-8
    LANGUAGE \(unset\)
    LD\_LIBRARY\_PATH=/usr/local/lib
    LOGDIR \(unset\)
    PATH=/usr/local/bin&#8203;:/usr/bin&#8203;:/cygdrive/c/Program Files/Target Compiler Technologies/DSP5\.0\.0\.0\-i32\-12R2\.6/bin/WINbin&#8203;:/cygdrive/h/TQH/bin&#8203;:/cygdrive/c/cs\-script&#8203;:/cygdrive/h/TQH/mksnt\.520/mksnt&#8203;:/cygdrive/h/TQH/bat&#8203;:/cygdrive/h/Program Files/Araxis/Araxis Merge&#8203;:/cygdrive/c/Program Files/vim/vim73&#8203;:/cygdrive/c/Program Files/Target Compiler Technologies/coyote5\-i32\-12R2/bin/WINbin&#8203;:/cygdrive/c/Program Files/GnuWin32/bin&#8203;:/cygdrive/c/MinGW/bin&#8203;:/cygdrive/c/Program Files/Target Compiler Technologies/coyote5\-12R1/bin/WINbin&#8203;:/cygdrive/c/Program Files/PC Connectivity Solution&#8203;:/cygdrive/c/program files/automated qa/testcomplete 8/bin&#8203;:/cygdrive/c/Program Files/CSR/BlueSuite 2\.5\.0/DFU Tools&#8203;:/cygdrive/c/Program Files/CSR/BlueSuite 2\.4/DFU Tools&#8203;:/cygdrive/c/Program Files/CSR/BlueSuite/DFU
 Tools&#8203;:/cygdrive/c/Perl/site/bin&#8203;:/cygdrive/c/Perl/bin&#8203;:/cygdrive/c/Tcl/bin&#8203;:/cygdrive/c/WINDOWS/system32&#8203;:/cygdrive/c/WINDOWS&#8203;:/cygdrive/c/WINDOWS/System32/Wbem&#8203;:/cygdrive/c/WINDOWS/system32/WindowsPowerShell/v1\.0&#8203;:/cygdrive/c/Program Files/Common Files/Lenovo&#8203;:/cygdrive/c/Program Files/Microsoft SQL Server/90/Tools/binn&#8203;:/cygdrive/c/Program Files/Microsoft SQL Server/100/Tools/Binn&#8203;:/cygdrive/c/Program Files/Microsoft SQL Server/100/DTS/Binn&#8203;:/cygdrive/c/Program Files/IBM/RationalSDLC/ClearCase/bin&#8203;:/cygdrive/c/Program Files/IBM/RationalSDLC/common&#8203;:/cygdrive/c/Program Files/Windows Imaging&#8203;:/cygdrive/c/Program Files/IVI Foundation/IVI/bin&#8203;:/cygdrive/c/Program Files/IVI Foundation/VISA/WinNT/Bin&#8203;:/cygdrive/c/Program Files/IVI Foundation/VISA/WinNT/Bin&#8203;:/cygdrive/c/Program Files/MATLAB/R2009a/bin&#8203;:/cygdrive/c/Program Files/MATLAB/R2009a/bin/win32&#8203;:/cygdrive/c/Program Files/Calibre2&#8203;:/cygdrive/c/Program Files/Microchip/MPLAB C32 Suite/bin&#8203;:/cygdrive/c/Program
 Files/Sysinternals Suite&#8203;:/cygdrive/c/Program Files/Universal Extractor&#8203;:/cygdrive/c/Program Files/Universal Extractor/bin&#8203;:/cygdrive/c/Program Files/Windows Installer XML v3\.5/bin&#8203;:/cygdrive/c/Program Files/GN ReSound/Coyote Assembler and Linker/bin&#8203;:/cygdrive/c/Program Files/Gallio/bin&#8203;:/cygdrive/c/Program Files/SlikSvn/bin&#8203;:/cygdrive/h/TQH/bin&#8203;:/cygdrive/c/Program Files/hdparm/bin&#8203;:/usr/lib/lapack
    PERL\_BADLANG \(unset\)
    SHELL=/bin/bash

**Comment by p5pRT** [comment]
### From @rurban

Please do not perlbug cpan bugs\. 
perlbug is only for problems with perl core and extensions 
which are handled by p5p\, and not the cpan author upstream\.

Please file your report at https://rt.cpan.org/Public/Dist/Display.html?Name=Sys-CPU

For the author&#8203;: processor\_info\_t is not supported on cygwin and is not portable\. 
Read /proc/cpuinfo as everyone else does\.
\-\- 
Reini Urban

**Comment by p5pRT** [comment]
The RT System itself - Status changed from 'new' to 'open'

**Comment by p5pRT** [comment]
@chorny - Status changed from 'open' to 'rejected'

**Comment by p5pRT** [comment]
### From @khwilliamson

On 07/09/2013 04&#8203;:55 AM\, Tuyen Hoang \(via RT\) wrote&#8203;:
> \# New Ticket Created by  Tuyen Hoang
> \# Please include the string&#8203;:  \[perl \#118819\]
> \# in the subject line of all future correspondence about this issue\.
> \# \<URL&#8203;: https://rt-archive.perl.org/perl5/Ticket/Display.html?id=118819 >
>
>
> Subject&#8203;: "cpan \-i Sys&#8203;::CPU" failed on Cygwin
> Message\-Id&#8203;: \<5\.14\.2\_19888\_1373366884@&#8203;AZHA1>
> Reply\-To&#8203;: tuyenqh@&#8203;yahoo\.com
> To&#8203;: perlbug@&#8203;perl\.org
> From&#8203;: tuyenqh@&#8203;yahoo\.com
>
>
> This is a bug report for perl from tuyenqh@&#8203;yahoo\.com\,
> generated with the help of perlbug 1\.39 running under perl 5\.14\.2\.
>
> On my Cyswin
>
> cpan \-i Sys&#8203;::CPU failed with following output
> The gcc version is 4\.7\.3
>
> Regards\,
> Tuyen

It appears that there is no appropriate Danish locale installed on your 
system\.  If you unset the environment variable LANG\, this error should 
go away\.


> =======================================================================
>
>
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> CPAN&#8203;: Storable loaded ok \(v2\.27\)
> Going to read '/home/tqhoang/\.cpan/Metadata'
>    Database was generated on Mon\, 08 Jul 2013 06&#8203;:07&#8203;:29 GMT
> Running install for module 'Sys&#8203;::CPU'
> Running make for M/MZ/MZSANFORD/Sys\-CPU\-0\.54\.tar\.gz
> CPAN&#8203;: Digest&#8203;::SHA loaded ok \(v5\.71\)
> CPAN&#8203;: Compress&#8203;::Zlib loaded ok \(v2\.033\)
> Checksum for /home/tqhoang/\.cpan/sources/authors/id/M/MZ/MZSANFORD/Sys\-CPU\-0\.54\.tar\.gz ok
> CPAN&#8203;: File&#8203;::Temp loaded ok \(v0\.22\)
> CPAN&#8203;: Parse&#8203;::CPAN&#8203;::Meta loaded ok \(v1\.4401\)
> CPAN&#8203;: CPAN&#8203;::Meta loaded ok \(v2\.110440\)
> CPAN&#8203;: Module&#8203;::CoreList loaded ok \(v2\.49\_02\)
>
>    CPAN\.pm&#8203;: Going to build M/MZ/MZSANFORD/Sys\-CPU\-0\.54\.tar\.gz
>
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> Checking if your kit is complete\.\.\.
> Looks good
> Writing Makefile for Sys&#8203;::CPU
> Writing MYMETA\.yml
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> cp CPU\.pm blib/lib/Sys/CPU\.pm
> /usr/bin/perl\.exe /usr/lib/perl5/5\.14/ExtUtils/xsubpp  \-typemap /usr/lib/perl5/5\.14/ExtUtils/typemap  CPU\.xs > CPU\.xsc && mv CPU\.xsc CPU\.c
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> Please specify prototyping behavior for CPU\.xs \(see perlxs manual\)
> gcc\-4 \-c    \-DPERL\_USE\_SAFE\_PUTENV \-U\_\_STRICT\_ANSI\_\_ \-g \-fno\-strict\-aliasing \-pipe \-fstack\-protector \-DUSEIMPORTLIB \-O3    \-DVERSION=\\"0\.54\\"  \-DXS\_VERSION=\\"0\.54\\"  "\-I/usr/lib/perl5/5\.14/i686\-cygwin\-threads\-64int/CORE"   CPU\.c
> CPU\.xs&#8203;: In function 'XS\_Sys\_\_CPU\_cpu\_clock'&#8203;:
> CPU\.xs&#8203;:349&#8203;:5&#8203;: error&#8203;: unknown type name 'processor\_info\_t'
> CPU\.xs&#8203;:350&#8203;:48&#8203;: error&#8203;: request for member 'pi\_state' in something not a structure or union
> CPU\.xs&#8203;:350&#8203;:62&#8203;: error&#8203;: 'P\_ONLINE' undeclared \(first use in this function\)
> CPU\.xs&#8203;:350&#8203;:62&#8203;: note&#8203;: each undeclared identifier is reported only once for each function it appears in
> CPU\.xs&#8203;:351&#8203;:26&#8203;: error&#8203;: request for member 'pi\_clock' in something not a structure or union
> CPU\.xs&#8203;:352&#8203;:26&#8203;: error&#8203;: request for member 'pi\_clock' in something not a structure or union
> CPU\.xs&#8203;: In function 'XS\_Sys\_\_CPU\_cpu\_type'&#8203;:
> CPU\.xs&#8203;:389&#8203;:5&#8203;: error&#8203;: unknown type name 'processor\_info\_t'
> CPU\.xs&#8203;:391&#8203;:16&#8203;: error&#8203;: request for member 'pi\_processor\_type' in something not a structure or union
> Makefile&#8203;:329&#8203;: recipe for target \`CPU\.o' failed
> make&#8203;: \*\*\* \[CPU\.o\] Error 1
>    MZSANFORD/Sys\-CPU\-0\.54\.tar\.gz
>    /usr/bin/make \-\- NOT OK
> 'YAML' not installed\, will not store persistent state
> Running make test
>    Can't test without successful make
> Running make install
>    Make had returned bad status\, install seems impossible
> \-\-\-
> Flags&#8203;:
>      category=library
>      severity=low
> \-\-\-
> Site configuration information for perl 5\.14\.2&#8203;:
>
> Configured by rurban at Thu Jul 12 13&#8203;:58&#8203;:56 CDT 2012\.
>
> Summary of my perl5 \(revision 5 version 14 subversion 2\) configuration&#8203;:
>
>    Platform&#8203;:
>      osname=cygwin\, osvers=1\.7\.15\(0\.26053\)\, archname=cygwin\-thread\-multi\-64int
>      uname='cygwin\_nt\-5\.1 winxp 1\.7\.15\(0\.26053\) 2012\-05\-09 10&#8203;:25 i686 cygwin '
>      config\_args='\-de \-Dlibperl=cygperl5\_14\.dll \-Dcc=gcc\-4 \-Dld=g\+\+\-4 \-Darchname=i686\-cygwin\-threads\-64int \-Dmksymlinks \-Dusethreads \-Accflags=\-g'
>      hint=recommended\, useposix=true\, d\_sigaction=define
>      useithreads=define\, usemultiplicity=define
>      useperlio=define\, d\_sfio=undef\, uselargefiles=define\, usesocks=undef
>      use64bitint=define\, use64bitall=undef\, uselongdouble=undef
>      usemymalloc=n\, bincompat5005=undef
>    Compiler&#8203;:
>      cc='gcc\-4'\, ccflags ='\-DPERL\_USE\_SAFE\_PUTENV \-U\_\_STRICT\_ANSI\_\_ \-g \-fno\-strict\-aliasing \-pipe \-fstack\-protector'\,
>      optimize='\-O3'\,
>      cppflags='\-DPERL\_USE\_SAFE\_PUTENV \-U\_\_STRICT\_ANSI\_\_ \-g \-fno\-strict\-aliasing \-pipe \-fstack\-protector'
>      ccversion=''\, gccversion='4\.5\.3'\, gccosandvers=''
>      intsize=4\, longsize=4\, ptrsize=4\, doublesize=8\, byteorder=12345678
>      d\_longlong=define\, longlongsize=8\, d\_longdbl=define\, longdblsize=12
>      ivtype='long long'\, ivsize=8\, nvtype='double'\, nvsize=8\, Off\_t='off\_t'\, lseeksize=8
>      alignbytes=8\, prototype=define
>    Linker and Libraries&#8203;:
>      ld='g\+\+\-4'\, ldflags =' \-Wl\,\-\-enable\-auto\-import \-Wl\,\-\-export\-all\-symbols \-Wl\,\-\-enable\-auto\-image\-base \-fstack\-protector \-L/usr/local/lib'
>      libpth=/usr/local/lib /usr/lib /lib
>      libs=\-lgdbm \-ldb \-ldl \-lcrypt \-lgdbm\_compat
>      perllibs=\-ldl \-lcrypt
>      libc=/usr/lib/libc\.a\, so=dll\, useshrplib=true\, libperl=cygperl5\_14\.dll
>      gnulibc\_version=''
>    Dynamic Linking&#8203;:
>      dlsrc=dl\_dlopen\.xs\, dlext=dll\, d\_dlsymun=undef\, ccdlflags=' '
>      cccdlflags=' '\, lddlflags=' \-\-shared  \-Wl\,\-\-enable\-auto\-import \-Wl\,\-\-export\-all\-symbols \-Wl\,\-\-enable\-auto\-image\-base \-L/usr/local/lib \-fstack\-protector'
>
> Locally applied patches&#8203;:
>      Bug\#55162 File&#8203;::Spec&#8203;::case\_tolerant performance
>      CYG07 $vendorarch/auto/\.rebase
>      CYG15 static Win32CORE
>      CYG17 cyg\-1\.7 paths\-utf8
>      0c612ce82 Fix building static extensions on cygwin\, \-UUSEIMPORTLIB
>      1bac5ecc1 Fix 64\-bit threading sv\.c&#8203;: S\_anonymise\_cv\_maybe
>      Cygwin&#8203;::sync\_winenv added
>
> \-\-\-
> @&#8203;INC for perl 5\.14\.2&#8203;:
>      /usr/lib/perl5/site\_perl/5\.14/i686\-cygwin\-threads\-64int
>      /usr/lib/perl5/site\_perl/5\.14
>      /usr/lib/perl5/vendor\_perl/5\.14/i686\-cygwin\-threads\-64int
>      /usr/lib/perl5/vendor\_perl/5\.14
>      /usr/lib/perl5/5\.14/i686\-cygwin\-threads\-64int
>      /usr/lib/perl5/5\.14
>      /usr/lib/perl5/site\_perl/5\.10
>      /usr/lib/perl5/vendor\_perl/5\.10
>      /usr/lib/perl5/site\_perl/5\.8
>      \.
>
> \-\-\-
> Environment for perl 5\.14\.2&#8203;:
>      HOME=/home/tqhoang
>      LANG=en\_US\.UTF\-8
>      LANGUAGE \(unset\)
>      LD\_LIBRARY\_PATH=/usr/local/lib
>      LOGDIR \(unset\)
>      PATH=/usr/local/bin&#8203;:/usr/bin&#8203;:/cygdrive/c/Program Files/Target Compiler Technologies/DSP5\.0\.0\.0\-i32\-12R2\.6/bin/WINbin&#8203;:/cygdrive/h/TQH/bin&#8203;:/cygdrive/c/cs\-script&#8203;:/cygdrive/h/TQH/mksnt\.520/mksnt&#8203;:/cygdrive/h/TQH/bat&#8203;:/cygdrive/h/Program Files/Araxis/Araxis Merge&#8203;:/cygdrive/c/Program Files/vim/vim73&#8203;:/cygdrive/c/Program Files/Target Compiler Technologies/coyote5\-i32\-12R2/bin/WINbin&#8203;:/cygdrive/c/Program Files/GnuWin32/bin&#8203;:/cygdrive/c/MinGW/bin&#8203;:/cygdrive/c/Program Files/Target Compiler Technologies/coyote5\-12R1/bin/WINbin&#8203;:/cygdrive/c/Program Files/PC Connectivity Solution&#8203;:/cygdrive/c/program files/automated qa/testcomplete 8/bin&#8203;:/cygdrive/c/Program Files/CSR/BlueSuite 2\.5\.0/DFU Tools&#8203;:/cygdrive/c/Program Files/CSR/BlueSuite 2\.4/DFU Tools&#8203;:/cygdrive/c/Program Files/CSR/BlueSuite/DFU
>   Tools&#8203;:/cygdrive/c/Perl/site/bin&#8203;:/cygdrive/c/Perl/bin&#8203;:/cygdrive/c/Tcl/bin&#8203;:/cygdrive/c/WINDOWS/system32&#8203;:/cygdrive/c/WINDOWS&#8203;:/cygdrive/c/WINDOWS/System32/Wbem&#8203;:/cygdrive/c/WINDOWS/system32/WindowsPowerShell/v1\.0&#8203;:/cygdrive/c/Program Files/Common Files/Lenovo&#8203;:/cygdrive/c/Program Files/Microsoft SQL Server/90/Tools/binn&#8203;:/cygdrive/c/Program Files/Microsoft SQL Server/100/Tools/Binn&#8203;:/cygdrive/c/Program Files/Microsoft SQL Server/100/DTS/Binn&#8203;:/cygdrive/c/Program Files/IBM/RationalSDLC/ClearCase/bin&#8203;:/cygdrive/c/Program Files/IBM/RationalSDLC/common&#8203;:/cygdrive/c/Program Files/Windows Imaging&#8203;:/cygdrive/c/Program Files/IVI Foundation/IVI/bin&#8203;:/cygdrive/c/Program Files/IVI Foundation/VISA/WinNT/Bin&#8203;:/cygdrive/c/Program Files/IVI Foundation/VISA/WinNT/Bin&#8203;:/cygdrive/c/Program Files/MATLAB/R2009a/bin&#8203;:/cygdrive/c/Program Files/MATLAB/R2009a/bin/win32&#8203;:/cygdrive/c/Program Files/Calibre2&#8203;:/cygdrive/c/Program Files/Microchip/MPLAB C32 Suite/bin&#8203;:/cygdrive/c/Program
>   Files/Sysinternals Suite&#8203;:/cygdrive/c/Program Files/Universal Extractor&#8203;:/cygdrive/c/Program Files/Universal Extractor/bin&#8203;:/cygdrive/c/Program Files/Windows Installer XML v3\.5/bin&#8203;:/cygdrive/c/Program Files/GN ReSound/Coyote Assembler and Linker/bin&#8203;:/cygdrive/c/Program Files/Gallio/bin&#8203;:/cygdrive/c/Program Files/SlikSvn/bin&#8203;:/cygdrive/h/TQH/bin&#8203;:/cygdrive/c/Program Files/hdparm/bin&#8203;:/usr/lib/lapack
>      PERL\_BADLANG \(unset\)
>      SHELL=/bin/bash
>
>
>
> Subject&#8203;: "cpan \-i Sys&#8203;::CPU" failed on Cygwin
> Message\-Id&#8203;: \<5\.14\.2\_19888\_1373366884@&#8203;AZHA1>
> Reply\-To&#8203;: tuyenqh@&#8203;yahoo\.com
> To&#8203;: perlbug@&#8203;perl\.org
> From&#8203;: tuyenqh@&#8203;yahoo\.com
>
>
> This is a bug report for perl from tuyenqh@&#8203;yahoo\.com\,
> generated with the help of perlbug 1\.39 running under perl 5\.14\.2\.
>
> On my Cyswin
>
> cpan \-i Sys&#8203;::CPU failed with following output
> The gcc version is 4\.7\.3
>
> Regards\,
> Tuyen
> =======================================================================
>
>
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> CPAN&#8203;: Storable loaded ok \(v2\.27\)
> Going to read '/home/tqhoang/\.cpan/Metadata'
>    Database was generated on Mon\, 08 Jul 2013 06&#8203;:07&#8203;:29 GMT
> Running install for module 'Sys&#8203;::CPU'
> Running make for M/MZ/MZSANFORD/Sys\-CPU\-0\.54\.tar\.gz
> CPAN&#8203;: Digest&#8203;::SHA loaded ok \(v5\.71\)
> CPAN&#8203;: Compress&#8203;::Zlib loaded ok \(v2\.033\)
> Checksum for
> /home/tqhoang/\.cpan/sources/authors/id/M/MZ/MZSANFORD/Sys\-CPU\-0\.54\.tar\.gz ok
> CPAN&#8203;: File&#8203;::Temp loaded ok \(v0\.22\)
> CPAN&#8203;: Parse&#8203;::CPAN&#8203;::Meta loaded ok \(v1\.4401\)
> CPAN&#8203;: CPAN&#8203;::Meta loaded ok \(v2\.110440\)
> CPAN&#8203;: Module&#8203;::CoreList loaded ok \(v2\.49\_02\)
>
>    CPAN\.pm&#8203;: Going to build M/MZ/MZSANFORD/Sys\-CPU\-0\.54\.tar\.gz
>
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> Checking if your kit is complete\.\.\.
> Looks good
> Writing Makefile for Sys&#8203;::CPU
> Writing MYMETA\.yml
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> cp CPU\.pm blib/lib/Sys/CPU\.pm
> /usr/bin/perl\.exe /usr/lib/perl5/5\.14/ExtUtils/xsubpp  \-typemap
> /usr/lib/perl5/5\.14/ExtUtils/typemap  CPU\.xs > CPU\.xsc && mv CPU\.xsc CPU\.c
> perl&#8203;: warning&#8203;: Setting locale failed\.
> perl&#8203;: warning&#8203;: Please check that your locale settings&#8203;:
> LC\_ALL = \(unset\)\,
> LANG = "DA"
>      are supported and installed on your system\.
> perl&#8203;: warning&#8203;: Falling back to the standard locale \("C"\)\.
> Please specify prototyping behavior for CPU\.xs \(see perlxs manual\)
> gcc\-4 \-c    \-DPERL\_USE\_SAFE\_PUTENV \-U\_\_STRICT\_ANSI\_\_ \-g
> \-fno\-strict\-aliasing \-pipe \-fstack\-protector \-DUSEIMPORTLIB \-O3
>   \-DVERSION=\\"0\.54\\"  \-DXS\_VERSION=\\"0\.54\\"
>   "\-I/usr/lib/perl5/5\.14/i686\-cygwin\-threads\-64int/CORE"   CPU\.c
> CPU\.xs&#8203;: In function 'XS\_Sys\_\_CPU\_cpu\_clock'&#8203;:
> CPU\.xs&#8203;:349&#8203;:5&#8203;: error&#8203;: unknown type name 'processor\_info\_t'
> CPU\.xs&#8203;:350&#8203;:48&#8203;: error&#8203;: request for member 'pi\_state' in something not a
> structure or union
> CPU\.xs&#8203;:350&#8203;:62&#8203;: error&#8203;: 'P\_ONLINE' undeclared \(first use in this function\)
> CPU\.xs&#8203;:350&#8203;:62&#8203;: note&#8203;: each undeclared identifier is reported only once
> for each function it appears in
> CPU\.xs&#8203;:351&#8203;:26&#8203;: error&#8203;: request for member 'pi\_clock' in something not a
> structure or union
> CPU\.xs&#8203;:352&#8203;:26&#8203;: error&#8203;: request for member 'pi\_clock' in something not a
> structure or union
> CPU\.xs&#8203;: In function 'XS\_Sys\_\_CPU\_cpu\_type'&#8203;:
> CPU\.xs&#8203;:389&#8203;:5&#8203;: error&#8203;: unknown type name 'processor\_info\_t'
> CPU\.xs&#8203;:391&#8203;:16&#8203;: error&#8203;: request for member 'pi\_processor\_type' in
> something not a structure or union
> Makefile&#8203;:329&#8203;: recipe for target \`CPU\.o' failed
> make&#8203;: \*\*\* \[CPU\.o\] Error 1
>    MZSANFORD/Sys\-CPU\-0\.54\.tar\.gz
>    /usr/bin/make \-\- NOT OK
> 'YAML' not installed\, will not store persistent state
> Running make test
>    Can't test without successful make
> Running make install
>    Make had returned bad status\, install seems impossible
> \-\-\-
> Flags&#8203;:
>      category=library
>      severity=low
> \-\-\-
> Site configuration information for perl 5\.14\.2&#8203;:
>
> Configured by rurban at Thu Jul 12 13&#8203;:58&#8203;:56 CDT 2012\.
>
> Summary of my perl5 \(revision 5 version 14 subversion 2\) configuration&#8203;:
>    Platform&#8203;:
>      osname=cygwin\, osvers=1\.7\.15\(0\.26053\)\,
> archname=cygwin\-thread\-multi\-64int
>      uname='cygwin\_nt\-5\.1 winxp 1\.7\.15\(0\.26053\) 2012\-05\-09 10&#8203;:25 i686
> cygwin '
>      config\_args='\-de \-Dlibperl=cygperl5\_14\.dll \-Dcc=gcc\-4 \-Dld=g\+\+\-4
> \-Darchname=i686\-cygwin\-threads\-64int \-Dmksymlinks \-Dusethreads \-Accflags=\-g'
>      hint=recommended\, useposix=true\, d\_sigaction=define
>      useithreads=define\, usemultiplicity=define
>      useperlio=define\, d\_sfio=undef\, uselargefiles=define\, usesocks=undef
>      use64bitint=define\, use64bitall=undef\, uselongdouble=undef
>      usemymalloc=n\, bincompat5005=undef
>    Compiler&#8203;:
>      cc='gcc\-4'\, ccflags ='\-DPERL\_USE\_SAFE\_PUTENV \-U\_\_STRICT\_ANSI\_\_ \-g
> \-fno\-strict\-aliasing \-pipe \-fstack\-protector'\,
>      optimize='\-O3'\,
>      cppflags='\-DPERL\_USE\_SAFE\_PUTENV \-U\_\_STRICT\_ANSI\_\_ \-g
> \-fno\-strict\-aliasing \-pipe \-fstack\-protector'
>      ccversion=''\, gccversion='4\.5\.3'\, gccosandvers=''
>      intsize=4\, longsize=4\, ptrsize=4\, doublesize=8\, byteorder=12345678
>      d\_longlong=define\, longlongsize=8\, d\_longdbl=define\, longdblsize=12
>      ivtype='long long'\, ivsize=8\, nvtype='double'\, nvsize=8\,
> Off\_t='off\_t'\, lseeksize=8
>      alignbytes=8\, prototype=define
>    Linker and Libraries&#8203;:
>      ld='g\+\+\-4'\, ldflags =' \-Wl\,\-\-enable\-auto\-import
> \-Wl\,\-\-export\-all\-symbols \-Wl\,\-\-enable\-auto\-image\-base \-fstack\-protector
> \-L/usr/local/lib'
>      libpth=/usr/local/lib /usr/lib /lib
>      libs=\-lgdbm \-ldb \-ldl \-lcrypt \-lgdbm\_compat
>      perllibs=\-ldl \-lcrypt
>      libc=/usr/lib/libc\.a\, so=dll\, useshrplib=true\, libperl=cygperl5\_14\.dll
>      gnulibc\_version=''
>    Dynamic Linking&#8203;:
>      dlsrc=dl\_dlopen\.xs\, dlext=dll\, d\_dlsymun=undef\, ccdlflags=' '
>      cccdlflags=' '\, lddlflags=' \-\-shared  \-Wl\,\-\-enable\-auto\-import
> \-Wl\,\-\-export\-all\-symbols \-Wl\,\-\-enable\-auto\-image\-base \-L/usr/local/lib
> \-fstack\-protector'
>
> Locally applied patches&#8203;:
>      Bug\#55162 File&#8203;::Spec&#8203;::case\_tolerant performance
>      CYG07 $vendorarch/auto/\.rebase
>      CYG15 static Win32CORE
>      CYG17 cyg\-1\.7 paths\-utf8
>      0c612ce82 Fix building static extensions on cygwin\, \-UUSEIMPORTLIB
>      1bac5ecc1 Fix 64\-bit threading sv\.c&#8203;: S\_anonymise\_cv\_maybe
>      Cygwin&#8203;::sync\_winenv added
>
> \-\-\-
> @&#8203;INC for perl 5\.14\.2&#8203;:
>      /usr/lib/perl5/site\_perl/5\.14/i686\-cygwin\-threads\-64int
>      /usr/lib/perl5/site\_perl/5\.14
>      /usr/lib/perl5/vendor\_perl/5\.14/i686\-cygwin\-threads\-64int
>      /usr/lib/perl5/vendor\_perl/5\.14
>      /usr/lib/perl5/5\.14/i686\-cygwin\-threads\-64int
>      /usr/lib/perl5/5\.14
>      /usr/lib/perl5/site\_perl/5\.10
>      /usr/lib/perl5/vendor\_perl/5\.10
>      /usr/lib/perl5/site\_perl/5\.8
>      \.
>
> \-\-\-
> Environment for perl 5\.14\.2&#8203;:
>      HOME=/home/tqhoang
>      LANG=en\_US\.UTF\-8
>      LANGUAGE \(unset\)
>      LD\_LIBRARY\_PATH=/usr/local/lib
>      LOGDIR \(unset\)
>      PATH=/usr/local/bin&#8203;:/usr/bin&#8203;:/cygdrive/c/Program Files/Target
> Compiler
> Technologies/DSP5\.0\.0\.0\-i32\-12R2\.6/bin/WINbin&#8203;:/cygdrive/h/TQH/bin&#8203;:/cygdrive/c/cs\-script&#8203;:/cygdrive/h/TQH/mksnt\.520/mksnt&#8203;:/cygdrive/h/TQH/bat&#8203;:/cygdrive/h/Program
> Files/Araxis/Araxis Merge&#8203;:/cygdrive/c/Program
> Files/vim/vim73&#8203;:/cygdrive/c/Program Files/Target Compiler
> Technologies/coyote5\-i32\-12R2/bin/WINbin&#8203;:/cygdrive/c/Program
> Files/GnuWin32/bin&#8203;:/cygdrive/c/MinGW/bin&#8203;:/cygdrive/c/Program
> Files/Target Compiler
> Technologies/coyote5\-12R1/bin/WINbin&#8203;:/cygdrive/c/Program Files/PC
> Connectivity Solution&#8203;:/cygdrive/c/program files/automated
> qa/testcomplete 8/bin&#8203;:/cygdrive/c/Program Files/CSR/BlueSuite 2\.5\.0/DFU
> Tools&#8203;:/cygdrive/c/Program Files/CSR/BlueSuite 2\.4/DFU
> Tools&#8203;:/cygdrive/c/Program Files/CSR/BlueSuite/DFU
> Tools&#8203;:/cygdrive/c/Perl/site/bin&#8203;:/cygdrive/c/Perl/bin&#8203;:/cygdrive/c/Tcl/bin&#8203;:/cygdrive/c/WINDOWS/system32&#8203;:/cygdrive/c/WINDOWS&#8203;:/cygdrive/c/WINDOWS/System32/Wbem&#8203;:/cygdrive/c/WINDOWS/system32/WindowsPowerShell/v1\.0&#8203;:/cygdrive/c/Program
> Files/Common Files/Lenovo&#8203;:/cygdrive/c/Program Files/Microsoft SQL
> Server/90/Tools/binn&#8203;:/cygdrive/c/Program Files/Microsoft SQL
> Server/100/Tools/Binn&#8203;:/cygdrive/c/Program Files/Microsoft SQL
> Server/100/DTS/Binn&#8203;:/cygdrive/c/Program
> Files/IBM/RationalSDLC/ClearCase/bin&#8203;:/cygdrive/c/Program
> Files/IBM/RationalSDLC/common&#8203;:/cygdrive/c/Program Files/Windows
> Imaging&#8203;:/cygdrive/c/Program Files/IVI
> Foundation/IVI/bin&#8203;:/cygdrive/c/Program Files/IVI
> Foundation/VISA/WinNT/Bin&#8203;:/cygdrive/c/Program Files/IVI
> Foundation/VISA/WinNT/Bin&#8203;:/cygdrive/c/Program
> Files/MATLAB/R2009a/bin&#8203;:/cygdrive/c/Program
> Files/MATLAB/R2009a/bin/win32&#8203;:/cygdrive/c/Program
> Files/Calibre2&#8203;:/cygdrive/c/Program Files/Microchip/MPLAB C32
> Suite/bin&#8203;:/cygdrive/c/Program Files/Sysinternals
> Suite&#8203;:/cygdrive/c/Program Files/Universal Extractor&#8203;:/cygdrive/c/Program
> Files/Universal Extractor/bin&#8203;:/cygdrive/c/Program Files/Windows
> Installer XML v3\.5/bin&#8203;:/cygdrive/c/Program Files/GN ReSound/Coyote
> Assembler and Linker/bin&#8203;:/cygdrive/c/Program
> Files/Gallio/bin&#8203;:/cygdrive/c/Program
> Files/SlikSvn/bin&#8203;:/cygdrive/h/TQH/bin&#8203;:/cygdrive/c/Program
> Files/hdparm/bin&#8203;:/usr/lib/lapack
>      PERL\_BADLANG \(unset\)
>      SHELL=/bin/bash
>
